projects/infinityworks/github-exporter/Makefile

```diff
@@ -0,0 +1,68 @@
+REPO?=github-exporter
```

Comment:
I know we always follow this pattern, but today I wonder why we allow REPO or CLONE_URL to be overwritten. Sure, I might want to do that for some special dev testing, but it should really never happen under normal circumstances. In the dev case we could do something else.

Reply:
The intent is to mirror GH repos to an internal CodeCommit repo, and clone from there, that way we aren't reliant on GH being available for builds

Reply:
If we were to mirror these repos, we could point these make files to the mirrors when they are available